packages/fxa-auth-server/test/local/routes/validators.js

```diff
@@ -450,4 +450,130 @@ describe('lib/routes/validators:', () => {
       assert.ok(res.error);
     });
   });
+
+  describe('subscriptionsStripeSubscriptionValidator', () => {
+    const { subscriptionsStripeSubscriptionValidator: subject } = validators;
+    it('accepts incomplete subscription with empty items and null cancel_at', () => {
```

Comment:
Does "empty items" here in the description mean that `data.items` below should not have any objects in its list?  I do see an object.

Reply:
Yeah, this description is kind of a guess at what was failing in the validator. Turns out it wasn't an items data object with an empty list - the problem was that item is itself an object rather than a list.